docs/basic-reader-and-writer.md

```diff
@@ -366,4 +366,42 @@ altered and the set of Streams being read has therefore been changed.
 So that's it.  The simple HelloWorldReader loops, reading Events from a Stream
 until there are no more Events, and then the application terminates.
 
- 
+## ReaderGroup Notifications
+
+ReaderGroup api supports different types of notifications. The supported
+notifications are
+
+1. SegmentEvent Notification
+
+A segment event is triggered when the total number of segments managed by the
+ReaderGroup changes. During a scale operation segments can be split into
```

Comment:
As we are referring to the concept and not an object or class, make the following change: ReaderGroup -> reader group.

Reply:
done.